README.md

```diff
@@ -1,579 +1,283 @@
-available [here](examples/typescript).
-
-### Python
-
-Similar to `node`, path to handler file `src/testing/handler.py`:
-
-```yml
-- src
-  - handlers
-  - firstHandler.py  => def my_first_handler
-  - secondHandler.py => def my_second_handler
-- serverless.yml
-```
-
-In serverless.yml:
-
-```yml
-provider:
-  # ...
-  runtime: python310 # or python37, python38, python39
-functions:
-  first:
-    handler: src/handlers/firstHandler.my_first_handler
-  second:
-    handler: src/handlers/secondHandler.my_second_handler
-```
-
-### Golang
-
-Path to your handler's **package**. For example, if you have the following structure:
-
-```yml
-- src
-  - testing
-  - handler.go -> package main in src/testing subdirectory
-  - second
-  - handler.go -> package main in src/second subdirectory
-- serverless.yml
-- handler.go -> package main at the root of project
-```
-
-Your serverless.yml `functions` should look something like this:
-
-```yml
-provider:
-  # ...
-  runtime: go118
-functions:
-  main:
-    handler: "."
-  testing:
-    handler: src/testing
-  second:
-    handler: src/second
-```
+    # Memory limit for the function (in MiB)
+    # Limits: https://www.scaleway.com/en/docs/serverless/functions/reference-content/functions-limitations/
+    memoryLimit: 1024
 
-### PHP
+    # Maximum duration a request will wait to be served before it times out (in seconds)
+    # Value in string format ex: "300s" (default: 300 seconds)
+    timeout: 300s
 
-Recommended folder structure for `php` runtimes:
+    # Runtime for this function, allows overriding provider.runtime
+    runtime: node20
 
-```yml
-├── handler.php
-├── composer.json (not necessary if you do not need dependencies)
-└── serverless.yml
-```
+    # Whether to permit HTTP or HTTPS-only. Options: enabled, disabled, or redirected
+    httpOption: enabled
 
-Your serverless.yml `functions` should look something like this:
+    # Whether the container is public (no authentication), or private (uses token-based authentication)
+    privacy: public
 
-```yml
-provider:
-  runtime: php82
-functions:
-  main:
-    handler: "handler"
-```
+    # Local environment variables, used only in this function
+    env:
+      LOCAL_VAR: "local var"
 
-### Rust
+    # Local secrets, used only in this function
+    secret:
+      LOCAL_SECRET: "local secret"
 
-Recommended folder structure for `rust` runtimes:
+    # Custom domains configured for the function
+    # https://www.scaleway.com/en/docs/compute/functions/how-to/add-a-custom-domain-name-to-a-function/
+    custom_domains:
+      - my-func.some.domain.com
+
+    # List of events to trigger the function
+    events:
+      - schedule:
+          rate: "1 * * * *"
+          # Data passed as input in the request
+          input:
+            key-a: "value-a"
+            key-b: "value-b"
+
+# Define containers, cannot be used with functions
+custom:
```

Comment:
maybe separate both files ? It can be impressive to see a huge configuration file